Framework/DataHandling/src/LoadAsciiStl.cpp

```diff
@@ -16,6 +16,10 @@ namespace DataHandling {
 
 bool LoadAsciiStl::isAsciiSTL() const {
   std::ifstream file(m_filename.c_str());
+  if (!file) {
```

Comment:
This is better, but there is a temporal coupling issue here that I don't like much. 
What you currently have:
```cpp
LoadAsciiStl mAlg;
....
myAlg.isAsciiSTL(); // returns false, but can happen because I forgot to call myAlg.setProperty("Filename", ...);
```
What would be better:
```cpp
LoadAsciiStl::isAsciiSTL(filename); // No temporal coupling. Don't have to know anything else about the rest of the class API to use. Compile time error if I forget to pass argument (yes, you could still pass it an default initialised string, but you already check for that.)
```